src/test/java/org/apache/sling/auth/oauth_client/OAuthUrisTest.java

```diff
@@ -44,17 +44,4 @@ void testRedirectUri() {
             .hasQuery("c=mock-oidc&redirect=/foo");
     }
 
-    @Test
-    void testRedirectUri_customPort_noRedirect() {
-        context.request().setServerPort(8080);
-        URI redirectUri = OAuthUris.getOAuthEntryPointUri(MockOidcConnection.DEFAULT_CONNECTION, context.request(), null);
```

Comment:
Would it make sense to pass an empty redirect URI instead? I think that is still possible.

Reply:
It can happens. Should `OAuthUris.getOAuthEntryPointUri()` return an exception in that case?
And in that case shall we propagate that exception in `OAuthEnabledSlingServlet` so that a 500 error with proper description is returned to the browser?

Reply:
No, I think it's fine for now to keep the current behaviour.